src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs

```diff
@@ -289,44 +286,39 @@ public void Given_TheNodeHadAReorg_And_WalletTipIsBehindConsensusTip_When_ANewBl
 
                 stratisSender.GenerateStratisWithMiner(10);
 
-                // wait for block repo for block sync to work
-                TestHelper.WaitLoop(() => TestHelper.IsNodeSynced(stratisSender));
-
-                //// sync all nodes
+                // Sync all nodes
```

Comment:
Is it not better to wait for one node to be fully synced, before making sure all the nodes are in sync with each other?